web/v1/redshift/sql-runner/sql/custom/01-indicative-export/02-indicative-export-staged.sql

```diff
@@ -0,0 +1,166 @@
+              e.refr_urlfragment,
+              e.refr_urlhost,
+              e.refr_urlpath,
+              e.refr_urlquery,
+              e.se_action,
+              e.se_category,
+              e.se_label,
+              e.se_property,
+              e.se_value,
+              e.ti_category,
+              e.ti_currency,
+              e.ti_name,
+              e.ti_orderid,
+              e.ti_price,
+              e.ti_price_base,
+              e.ti_quantity,
+              e.ti_sku,
+              e.tr_affiliation,
+              e.tr_city,
+              e.tr_country,
+              e.tr_currency,
+              e.tr_orderid,
+              e.tr_shipping,
+              e.tr_shipping_base,
+              e.tr_state,
+              e.tr_tax,
+              e.tr_tax_base,
+              e.tr_total,
+              e.tr_total_base,
+              e.user_fingerprint,
+              e.user_id,
+              e.user_ipaddress,
+              e.useragent,
+              e.collector_tstamp,
+              e.event_name,
+              CASE
+                  WHEN event_name = 'page_view' AND page_urlpath = '/' THEN 'homepage_view'
+                  ELSE event_name
+              END AS custom_event_name, -- example derived event name field
+              e.derived_tstamp
+          FROM {{.atomic_schema}}.events_staged{{.entropy}} AS e
+          WHERE app_id = 'website'),
+
+        child_table_joins AS
+         (SELECT
+              b.*,
+              cf.form_id AS change_form_form_id,      -- change form fields
+              cf.element_id AS change_form_element_id,
+              cf.node_name AS change_form_node_name,
+              cf.element_classes AS change_form_element_classes,
+              sf.form_id AS submit_form_form_id,      -- submit form fields
+              sf.form_classes AS submit_form_form_classes,
+              sf.elements AS submit_form_elements,
+              lc.element_id AS link_click_element_id, -- link click fields
+              lc.element_classes AS link_click_element_classes,
+              lc.element_target AS link_click_element_target,
+              lc.target_url AS link_click_target_url,
+              wsf.email AS sign_up_email,             -- website sign up form fields
+              wsf.company AS sign_up_company,
+              wsf.serviceType AS sign_up_service_type,
+              dr.company AS demo_request_company,     -- demo request fields
+              dr.email AS demo_request_email,
+              dr.insights AS demo_request_insights,
+              wp.id AS page_view_id
+          FROM base_table AS b
+                   LEFT JOIN {{.atomic_schema}}.com_snowplowanalytics_snowplow_change_form_1 AS cf
+          ON cf.root_id = b.event_id -- add any necessary joins into your query and the necessary columns
+              AND cf.root_tstamp = b.collector_tstamp
+              LEFT JOIN {{.atomic_schema}}.com_snowplowanalytics_snowplow_submit_form_1 AS sf ON sf.root_id = b.event_id
```

Comment:
I think as an example performance isn't massively critical. I'll check with Cara, as we're going to write a blog post with the standalone as a query, then an example custom module (to answer your earlier question), so we might want the custom module to be as performant as possible, or it might not be an issue. I don't have strong feelings either way either!